changelog/unreleased/issue-19339.toml

```diff
@@ -0,0 +1,5 @@
+type="a"
+message="Migrate export widget feture to enterprise"
+
+issues=["19339", "graylog-plugin-enterprise#7272"]
+pulls=["19340", "graylog-plugin-enterprise#7273"]
```

Comment:
There is a typo in `feture`.

Since this feature has never release, wouldn't it make sense to just move change log in https://github.com/Graylog2/graylog2-server/pull/19140 to the enterprise repo and not add a change log in this PR?

Reply:
This sounds like we are removing the message widget export feature from open source.

Reply:
I think we should just remove the changelog snippet in core.

Reply:
We should also remove `issue-19139.toml`.

Reply:
@dennisoelkers About issue-19139.toml we moved the messages export action to icon. Maybe it makes sense to keep at least  that ?

Reply:
That makes sense, yes!